src/PSR7/Validators/BodyValidator/BodyValidator.php

```diff
@@ -65,11 +65,11 @@ public function validate(OperationAddress $addr, MessageInterface $message) : vo
 
         // Validate message body
         if (preg_match('#^multipart/.*#', $contentType)) {
-            (new MultipartValidator($mediaTypeSpecs[$contentType], $contentType))->validate($addr, $message);
+            (new MultipartValidator($mediaTypeSpec, $contentType))->validate($addr, $message);
```

Comment:
Can we add more tests on this line to be sure they are behave correctly using range?

Reply:
Do we need this because the issue was not because of the strict comparison as i mentioned above 🤔

Reply:
Ah, got it